app/backend/src/ui/methods/task/update-task.ts

```diff
@@ -10,5 +11,20 @@ import TaskModel from '../../../database/models/task';
 export async function updateTask(data: UpdateTaskPayload): Promise<Task | null> {
   const query = { _id: data._id };
 
-  return TaskModel.findOneAndUpdate(query, data).exec();
+  const task = await TaskModel.findOneAndUpdate(query, data, { new: true }).exec();
+
+  if (!task || !task.statusId) {
```

Comment:
please describe the case. When the `null` could be returned?

Reply:
condition was wrong. `null` can be returned only when there is no task found